File: 0001 Code Chef/0017 Healthy Sleep/implementation/0010 nodejs/healthy_sleep.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let sleep = parseInt(input);
    if(sleep == 8){
        console.log("PERFECT");
    }else if (sleep < 8){
        console.log("LESS");
    }else if (sleep > 8){
        console.log("MORE");
    }else{
        console.log("Invalid input");
    }
    rl.close();
})

File: 0001 Code Chef/0017 Healthy Sleep/implementation/0005 kotlin/HealthySleep.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
           var sleep:Int? = stringInput.toIntOrNull();
           if(sleep != null){
                if(sleep == 8){
                    println("PERFECT");
                }else if (sleep < 8){
                    println("LESS");
                }else if (sleep > 8){
                    println("MORE");
                }else{
                    println("Invalid Input");
                }
           }
    }
}